src/as-validator-issue-tag.h

```diff
@@ -870,6 +870,11 @@ AsValidatorIssueTag as_validator_issue_tag_list[] =  {
 	  N_("The release timestamp is invalid."),
 	},
 
+	{ "release-description-outside-tag",
+	  AS_ISSUE_SEVERITY_ERROR,
+	  N_("The release description must be put inside a description tag"),
```

Comment:
```suggestion
	  /* TRANSLATORS: Please do not translate AppStream tag and property names (in backticks). */
	  N_("The release description must be put inside a `description` tag"),
```

Translators might translate this to something odd otherwise ;-)